cudaaligner/src/hirschberg_myers_gpu.cu

```diff
@@ -344,7 +344,7 @@ myers_compute_scores(
             }
             if (threadIdx.x == 0)
                 warp_carry = 0;
-            if (warp_mask == 0xffff'ffffu)
+            if (warp_mask == 0xffff'ffffu && (threadIdx.x == 0 || threadIdx.x == 31))
```

Comment:
are these bug fixes to the myers implementation? did these cause deadlocks?

Reply:
No, this was an oversight. It is not related to fixing the deadlocks I found.